contracts/PepemonMatchmaker.sol

```diff
@@ -151,10 +167,12 @@ contract PepemonMatchmaker is ERC1155Holder, ERC721Holder, Ownable {
     function processMatch(uint256 player1deckId, uint256 player2deckId) internal {
         // Evaluate the battle winner
         (address winner, address loser, uint256 battleId) = doBattle(player1deckId, player2deckId);
-        // Send a reward to the winner
-        RewardPool(_rewardPoolAddress).sendReward(battleId, winner);
         // Declare loser and winner
         emit BattleFinished(winner, loser, battleId);
+
+        // TODO: uncomment this before deploying to mainnet, this logic is important but makes testing difficult
+        // Send a reward to the winner
+        //RewardPool(_rewardPoolAddress).sendReward(battleId, winner);
```

Comment:
I think it's fine if the reward pool does nothing when there are no rewards left
That would be a bit easier to manage than commenting out stuff